env/env.go

```diff
@@ -33,25 +33,28 @@ func LoadEnv(config interface{}) error {
 		return err
 	}
 
-	// Format .env file keys.
-	for k, v := range dotEnvMap {
-		delete(dotEnvMap, k)
-		if strings.HasPrefix(k, goconfig.PrefixEnv) {
-			k = strings.TrimPrefix(k, goconfig.PrefixEnv+"_")
-		}
-		dotEnvMap[strings.ToLower(k)] = v
-	}
-
 	configType := reflect.TypeOf(config).Elem()
 	configValue := reflect.ValueOf(config).Elem()
 
 	for i := 0; i < configType.NumField(); i++ {
 		field := configType.Field(i)
-		tag := field.Tag.Get("cfg")
-		if tag == "" {
+
+		confKey := field.Tag.Get("env")
```

Comment:
nit: could this be extracted to a function and does that make sense? It seems really similar to code in `LoadEnv` function.

Reply:
Agree, extracting into separate function makes sense. Good point. Will also make sure the tag parsing priority has a single source of truth.